porting/building-halium-boot.rst

```diff
@@ -0,0 +1,44 @@
+Building halium-boot
+=====================
+
+Halium-boot is the new boot image in the Halium project, replacing hybris-boot. We will be building and using it for Ubuntu Touch.
```

Comment:
s/the new/a new proposed/

Reply:
Hey @UniversalSuperBox , we just discussed on tg. What about the following formulation:

Halium-boot is the new boot image intended for the Halium project. Ubuntu Touch uses halium-boot, other operating systems use hybris-boot at the moment.